### PR TITLE
feat(SPV-802): adjust pike config to new variables

### PR DESCRIPTION
**Pull request by pawellewandowski98**
# Pull Request Checklist

- [ ] 📖 I created my PR using provided: [CODE_STANDARDS](https://github.com/bitcoin-sv/spv-wallet-web-frontend/blob/main/.github/CODE_STANDARDS.md)
- [ ] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/spv-wallet-web-frontend/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [ ] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [ ] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

File: src/hooks/useFeatureFlags.ts [hunk 1]
```diff
@@ -4,6 +4,11 @@ export const useExperimentalFeatures = () => {
...
-export const usePikeEnabled = () => {
-  return useExperimentalFeatures().pike_enabled;
+export const usePikePaymentEnabled = () => {
+  return useExperimentalFeatures().pike_payment_enabled;
 };
+
+
+export const usePikeContactsEnabled = () => {
+  return useExperimentalFeatures().pike_contacts_enabled;
+};
```
Comment (chris-4chain) [inline src/hooks/useFeatureFlags.ts:14]: Here the end line is missing.

Comment (pawellewandowski98) [reply src/hooks/useFeatureFlags.ts]: Done

File: src/views/Dashboard/Dashboard.tsx [hunk 2]
```diff
@@ -4,13 +4,13 @@ import { AccountSummary } from '@/components/AccountSummary';
...
 import { useWebsocket } from '@/hooks';
-import { usePikeEnabled } from '@/hooks/useFeatureFlags';
+import { usePikePaymentEnabled } from '@/hooks/useFeatureFlags';
 
 export const Dashboard = () => {
   const lgMatch = useMediaMatch('lg');
   useWebsocket();
 
-  const pikeEnabled = usePikeEnabled();
+  const pikePaymentEnabled = usePikePaymentEnabled();
```
Comment (chris-4chain) [inline src/views/Dashboard/Dashboard.tsx:13]: It should be used `usePikeContactsEnabled();`

**Review by chris-4chain — CHANGES_REQUESTED** [review]

File: src/views/Dashboard/Dashboard.tsx (hunk 2 above)
Comment (pawellewandowski98) [reply src/views/Dashboard/Dashboard.tsx]: Done

**Review by chris-4chain — APPROVED** [review]

**Pull request merged**